src/webhooks/pubsub/slackScores.test.ts

```diff
@@ -0,0 +1,114 @@
+    postMessageSpy = jest.spyOn(bolt.client.chat, 'postMessage');
+  });
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should handle case when no issues are returned', async () => {
+    getIssueEventsForTeamSpy.mockReturnValue([]);
+    await triggerSlackScores(null, null);
+    expect(postMessageSpy).toHaveBeenCalledWith({
+      blocks: [
+        {
+          text: {
+            emoji: true,
+            text: '🗓️ Weekly GitHub Response Times by Team 🗓️',
+            type: 'plain_text',
+          },
+          type: 'header',
+        },
+        {
+          text: {
+            text: `\`\`\`┌────────────────────────────────────────────────┐
+| Team                          │ % on Time      |
+├────────────────────────────────────────────────┤
+| ospo                          | 100 (0/0)      |
+| enterprise                    | 100 (0/0)      |
+| issues                        | 100 (0/0)      |
+| ingest                        | 100 (0/0)      |
+| null                          | 100 (0/0)      |
+└────────────────────────────────────────────────┘\`\`\``,
+            type: 'mrkdwn',
+          },
+          type: 'section',
+        },
+      ],
+      channel: 'G01F3FQ0T41',
+      text: 'Weekly GitHub Team Scores',
+    });
+  });
+
+  it('should handle case when issues are returned and sorted by best response times', async () => {
+    getIssueEventsForTeamSpy
+      .mockReturnValueOnce([
+        {
+          issue_id: 1,
+          repository: 'routing-repo',
+          product_area: 'One-Team',
+          is_triaged: true,
+        },
+      ])
+      .mockReturnValueOnce([
+        {
+          issue_id: 2,
+          repository: 'routing-repo',
+          product_area: 'Multi-Team',
+          is_triaged: false,
+        },
+      ])
+      .mockReturnValueOnce([
+        {
+          issue_id: 3,
+          repository: 'routing-repo',
+          product_area: 'Test',
+          is_triaged: true,
+        },
+        {
+          issue_id: 4,
+          repository: 'routing-repo',
+          product_area: 'Test',
+          is_triaged: false,
+        },
+      ])
+      .mockReturnValue([]);
+    await triggerSlackScores(null, null);
+    expect(postMessageSpy).toHaveBeenCalledWith({
+      blocks: [
+        {
+          text: {
+            emoji: true,
+            text: '🗓️ Weekly GitHub Response Times by Team 🗓️',
+            type: 'plain_text',
+          },
+          type: 'header',
+        },
+        {
+          text: {
+            text: `\`\`\`┌────────────────────────────────────────────────┐
+| Team                          │ % on Time      |
+├────────────────────────────────────────────────┤
+| ospo                          | 100 (1/1)      |
+| issues                        | 50  (1/2)      |
+| enterprise                    | 0   (0/1)      |
+| ingest                        | 100 (0/0)      |
+| null                          | 100 (0/0)      |
```

Comment:
"null" is weird ... what does it mean? Would "other" or "no team" be clearer?

Reply:
It's the name I'm using to indicate that the `slack_channel` and `offices` fields in product-owners.yml is null. I don't think that other or no team would be clearer for that.

Reply:
Oh, hrm ... well I clearly didn't understand what it means! 😅

We can only know what team an issue is for if we have a `slack_channel` and `offices`?

Reply:
Not quite, it's a team I've used for other test suites. It's something that's useful to test to ensure eng-pipes doesn't break when slack notifications are being sent out to a team without a `slack_channel` or `offices`, but it's not useful for this PR at all. It's loaded in to the test suite on initialization which is why it appeared here. Does that make sense?

It's defined [here](https://github.com/getsentry/eng-pipes/blob/main/test/product-owners.yml#L42)

I have attempted to try and reload `product-owners.yml` for each test case with no luck and eventually gave up, so it's shared between test files right now 😔

Reply:
Ah! So `null` only shows up in test suites and not in prod?

Reply:
yep, it only appears in test suites

Reply:
Got it, thanks!